src/Expr.cc

```diff
@@ -3248,8 +3252,46 @@ RecordConstructorExpr::RecordConstructorExpr(ListExprPtr constructor_list)
 	SetType(make_intrusive<RecordType>(record_types));
 	}
 
-RecordConstructorExpr::~RecordConstructorExpr()
+RecordConstructorExpr::RecordConstructorExpr(RecordTypePtr known_rt,
+                                             ListExprPtr constructor_list)
+: Expr(EXPR_RECORD_CONSTRUCTOR)
```

Comment:
Initialization of `op` can be done in the initializer list.

Reply:
Done.